<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddNewFilm</class>
 <widget class="QDialog" name="AddNewFilm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add New Movie</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="0">
    <widget class="QLabel" name="label_duration">
     <property name="text">
      <string>Duration:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="lineEdit_title"/>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="lineEdit_type"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_type">
     <property name="text">
      <string>Type:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_genere">
     <property name="text">
      <string>Genere:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_year">
     <property name="text">
      <string>Year:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="lineEdit_year"/>
   </item>
   <item row="5" column="0">
    <widget class="QPushButton" name="pushButton_ok">
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_title">
     <property name="text">
      <string>Title:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="lineEdit_genre"/>
   </item>
   <item row="5" column="1">
    <widget class="QPushButton" name="pushButton_close">
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="lineEdit_duration"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_ok</sender>
   <signal>clicked()</signal>
   <receiver>AddNewFilm</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>48</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_close</sender>
   <signal>clicked()</signal>
   <receiver>AddNewFilm</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>242</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
